python/mxnet/image.py

```diff
@@ -89,7 +89,31 @@ def fixed_crop(src, x0, y0, w, h, size=None, interp=2):
 
 
 def random_crop(src, size, interp=2):
-    """Randomly crop src with size. Upsample result if src is smaller than size."""
+    """Randomly crop src with size. Upsample result if src is smaller than size.
+
+    Parameters
+    ----------
+    src : source ndarray
+    size : (width, height) - size of the crop. If it is larger than the image, then 
```

Comment:
Please start descriptions with capital letter and end with periods. Also `size` is the parameter name so it needs to appear in single backticks.

"Size of the crop, formatted as (width, height). If the desired size is larger than the image,
 then the image is upsampled to `size` and returned."